### DISCUSSION TITLE
Any plans to update Linux kernel version

### DISCUSSION THREAD

**Discussion by rcgeorge23** [bottlerocket-os/bottlerocket#2077] [Q&A] [answered]
Hi all,

I was wondering whether there were any plans to move bottlerocket to a more recent version of the Linux kernel.

I’m interested in running Docker with sysbox on bottlerocket, and I believe kernel version 5.12 is a prerequisite.

Thanks
Richard

**Comment by zmrow** [comment, marked as answer]
HI there!  We're planning to move the 5.15 kernel for all new variants that are going to be released this year, starting with the `*-k8s-1.23` variants in ~May.

https://github.com/bottlerocket-os/bottlerocket/issues/2008

**Comment by bcressey** [comment]
We won't quite be ready with 5.15 by the time the `*-k8s-1.23` variants land, so it's looking like it'll happen closer to August instead.

**Reply by vinyaMad** [reply]
Hello:

Do you have an update about this? I am also interested in using the 5.15 kernel

**Reply by etungsten** [reply]
Hi, we're currently planning to include the 5.15 kernel in the upcoming `*-k8s-1.24` variants (see https://github.com/bottlerocket-os/bottlerocket/issues/2374). The K8s 1.24 variants is currently slated to go out with our v1.10 Bottlerocket release in the later half of September.

**Comment by artyom-p** [comment]
Any plans to update to 5.18?

**Reply by stmcginnis** [reply]
Hi @artyom-p! There are no plans to use the 5.18 kernel, but [6.1](https://github.com/bottlerocket-os/bottlerocket/pull/3121) was recently added. That will be the kernel version used for the new [Kubernetes 1.28](https://github.com/bottlerocket-os/bottlerocket/pull/3329) and later variants, and the newer [ECS-2 variant](https://github.com/bottlerocket-os/bottlerocket/pull/3273).
